src/java/components/gpiomanager/src/test/java/hu/bme/mit/inf/modes3/components/gpiomanager/TestGpioReader.java

```diff
@@ -0,0 +1,14 @@
+package hu.bme.mit.inf.modes3.components.gpiomanager;
+
+import java.io.IOException;
+
+import hu.bme.mit.inf.modes3.components.gpiomanager.Gpio.Level;
+
+public class TestGpioReader implements CommandReader {
```

Comment:
Why is a separate class created for test, instead of using Mockito and creating a Mock in the test case itself?
If you just return the same static value every time, then a Mock might be more suitable instead of a generally named `TestGpioReader` class.

Reply:
True, it can be refactored.